index_V1

```diff
@@ -85,25 +217,62 @@
+					<div onclick="location.href='query_greatest.php';" class="circle_1" style="cursor: pointer; grid-column: 2; background-color: #e6e6e6; margin: 1vw 1vw; transition: 0.6s; overflow: hidden;">
+						
+						<img src="images/img_10.jpg" img draggable="false" alt="Blue" style="width: 12vw; height: 12vw;">
+					
+					</div>
 					
-					<div onclick="location.href='query_1.php';" class="circle_1" style="cursor: pointer; grid-column: 2; background-color: #e6e6e6; margin: 1vw 1vw; transition: 0.6s;"></div>
+					<div onclick="location.href='query_gym.php';" class="circle_2" style="cursor: pointer; grid-column: 3; background-color: #e6e6e6; margin: 1vw 1vw; transition: 0.6s; overflow: hidden;">
+						
+						<img src="images/img_13.jpg" img draggable="false" alt="Blue" style="width: 12vw; height: 12vw;">
 					
-					<div onclick="location.href='query_1.php';" class="circle_2" style="cursor: pointer; grid-column: 3; background-color: #e6e6e6; margin: 1vw 1vw; transition: 0.6s;"></div>
+					</div>
 					
-					<div onclick="location.href='query_1.php';" class="circle_3" style="cursor: pointer; grid-column: 4; background-color: #e6e6e6; margin: 1vw 1vw; transition: 0.6s;"></div>
+					<div onclick="location.href='query_love.php';" class="circle_3" style="cursor: pointer; grid-column: 4; background-color: #e6e6e6; margin: 1vw 1vw; transition: 0.6s; overflow: hidden;">
+						
+						<img src="images/img_11.jpg" img draggable="false" alt="Blue" style="width: 12vw; height: 12vw;">
 					
-					<div onclick="location.href='query_1.php';" class="circle_4" style="cursor: pointer; grid-column: 5; background-color: #e6e6e6; margin: 1vw 1vw; transition: 0.6s;"></div>
+					</div>
+					
+					<div onclick="location.href='query_road.php';" class="circle_4" style="cursor: pointer; grid-column: 5; background-color: #e6e6e6; margin: 1vw 1vw; transition: 0.6s; overflow: hidden;">
+						
+						<img src="images/img_12.jpg" img draggable="false" alt="Blue" style="width: 12vw; height: 12vw;">
+					
+					</div>
 				
 				</div>
 			
 			</div>
 				
 			</div>
 			
-			<div class="highlight"></div>
+			<div class="highlight" style="background-image: url('images/img_14.jpg'); background-size: 100vw 58vw;">
+			
+				<h1 style="margin-left: 48vw;">Placeholder.</h1>
+				<h2 style="margin-left: 58vw;">Lorem Ipsum.</h2>
+				
+				<a href="index.php#top"><img src="images/img_16.png" style="height: 36vw; width: 24vw; margin: -19.5vw 0vw 0vw 15vw;" draggable="false"></a>
+			
+			</div>
 			
-			<div class="footer"></div>
+			<div class="footer">
+				
+				
+				<h2 style="margin-left: 8vw; color: skyblue; margin-top: 3vw;">Author's Note.</h2>
+				<p style="color: white; margin-left: 8vw;">Suggested Playlists we recommend you to check out!<br>© Copyright EDevera 2021, All rights reserved.</p>
+			
+				<a href="index.php#top" class="bottom-to-top"><img src="images/icon_7.png" style="height: 7vw; width: 7vw; margin: -14vw 0vw 0vw 85vw;" draggable="false"></a>
+			
```

Comment:
Lastly, I updated this page by adding a basic bottom to top button as well as importing an icon to go along with it.